model/otlpgrpc/logs.go

```diff
@@ -133,7 +138,11 @@ func (lr LogsRequest) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON unmarshalls LogsRequest from JSON bytes.
 func (lr LogsRequest) UnmarshalJSON(data []byte) error {
```

Comment:
Unrelated to your change: looks like for some reason we now have 2 places in the codebase to unmarshal OTLP JSON. I am not sure if this duplication is intentional.